tests/templates/kuttl/ldap-authentication/07-assert.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: kubectl exec -n $NAMESPACE checks-0 -- python /tmp/authcheck.py
+timeout: 600
```

Comment:
This is a little much of a timeout for this check? Will extend the test feedback loop about 10minutes if anything fails here? Is this really necessary?